From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexandru Csete <oz9aec@gmail.com>
Date: Sat, 19 Jan 2019 16:26:35 +0100
Subject: [PATCH] Require Glib 2.40 or later

Issue #34.

Clear Linux packaging note:

 - Removed diff to NEWS due to conflicts

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index e3fe564..058a57e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -36,12 +36,12 @@ else
     AC_MSG_ERROR(Gpredict requires libcurl-dev 7.19 or later)
 fi
 
-# check for glib >2.32
-if pkg-config --atleast-version=2.32 glib-2.0; then
+# check for glib 2.40 or later
+if pkg-config --atleast-version=2.40 glib-2.0; then
     CFLAGS="$CFLAGS"
     LIBS="$LIBS"
 else
-    AC_MSG_ERROR(Gpredict requires libglib-dev 2.32 or later)
+    AC_MSG_ERROR(Gpredict requires libglib-dev 2.40 or later)
 fi
 
 # check for goocanvas (depends on gtk and glib)
